src/com/lftechnology/java/training/alina/basic/stemleaf/StemLeafRepresentation.java

```diff
@@ -0,0 +1,90 @@
+package com.lftechnology.java.training.alina.basic.stemleaf;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.InputMismatchException;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Java program to present data in stem and leaf form
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class StemLeafRepresentation {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)